### DISCUSSION TITLE
Hello from GitHub 👋

### DISCUSSION THREAD

**Discussion by rudy750** [apache/devlake#7642] [General]
Hey team,

A few of us are taking a look at your project and hope that you would be willing to chat with us. Let's set it up 👍

**Comment by d4x1** [comment]
@rudy750  Thank you for your message. Can you briefly explain your purpose? Let me see what we can do for you?

**Comment by rudy750** [comment]
We are a team that works directly with Enterprise customers. There is a lot of interest on metrics and dashboards to view productivity gains, DORA metrics etc..

We are hoping to get on a Zoom with the team to learn more about this project and how it can help our customers.

**Reply by d4x1** [reply]
@rudy750  That's fantastic news!
what's your timezone ? and  what time is good for you?

**Reply by rudy750** [reply]
Most of us are EST.

We are doing an initial review this Monday June 24th at 4 PM EST. You can join us then or we can set up another time

**Reply by hezyin** [reply]
Hi @rudy750, good to e-meet you, my name is Henry and I'm on DevLake PPMC. Happy to chat at 4 pm EST next Monday (June 24th). If you could share your emails, I'll send you a calendar invite. Looking forward to it!

**Reply by rudy750** [reply]
@hezyin 

Hey there 👋 

Great to meet you. We actually already have some time set aside for the meeting can we do it the other way around where we send you an invite? I can send to the email on your profile or is there another one you prefer

**Reply by hezyin** [reply]
@rudy750 Sure thing. Could you please send an invite to henry@merico.dev and anmol@merico.dev? Thank you.

**Reply by rudy750** [reply]
You have mail 📬
